docs/usage/querying.rst

```diff
@@ -0,0 +1,180 @@
+
+
+Get all entities of an entity set
+---------------------------------
+
+Print unique identification (Id) and last name of all employees:
+
+.. code-block:: python
+
+    employees = northwind.entity_sets.Employees.get_entities().select('EmployeeID,LastName').execute()
+    for employee in employees:
+        print(employee.EmployeeID, employee.LastName)
+
+
+Get entities matching a filter
+------------------------------
+
+Print unique identification (Id) of all employees with name John Smith:
+
+.. code-block:: python
+
+    smith_employees_request = northwind.entity_sets.Employees.get_entities()
+    smith_employees_request = smith_employees_request.filter("FirstName eq 'John' and LastName eq 'Smith'")
+
+    for smith in smith_employees.execute():
+        print(smith.EmployeeID)
+
+
+Get entities matching a filter in more Pythonic way
+---------------------------------------------------
+
+Print unique identification (Id) of all employees with name John Smith:
+
+.. code-block:: python
+
+    from pyodata.v2.service import GetEntitySetFilter as esf
+
+    smith_employees_request = northwind.entity_sets.Employees.get_entities()
+    smith_employees_request = smith_employees_request.filter(sef.and_(
+                                                             smith_employees_request.FirstName == 'Jonh',
+                                                             smith_employees_request.LastName == 'Smith'))
+    for smith in smith_employees_request.execute():
+        print(smith.EmployeeID)
+
+
+Get a count of entities
+-----------------------
+
+Print a count of all employees:
+
+.. code-block:: python
+
+    count = northwind.entity_sets.Employees.get_entities().count().execute()
+    print(count)
+
+
+Get a count of entities via navigation property
+-----------------------------------------------
+
+Print a count of all orders associated with Employee 1:
+
+.. code-block:: python
+
+    count = northwind.entity_sets.Employees.get_entity(1).nav('Orders').get_entities().count().execute()
+    print(count)
+
+
+Creating entity
+---------------
+
+You need to use the method set which accepts key value parameters:
+
+.. code-block:: python
+
+    employee_data = {
+        'FirstName': 'Mark',
+        'LastName': 'Goody',
+        'Address': {
+            'HouseNumber': 42,
+            'Street': 'Paradise',
+            'City': 'Heaven'
+        }
+    }
+
+    create_request = northwind.entity_sets.Employees.create_entity()
+    create_request.set(**employee_data)
+
+    new_employee_entity = create_request.execute()
+
+
+or you can do it explicitly:
+
+.. code-block:: python
+
+    create_request = northwind.entity_sets.Employees.create_entity()
+    create_request.set(
+        FirstName='Mark',
+        LastName='Goody',
+        Address={
+            'HouseNumber': 42,
+            'Street': 'Paradise',
+            'City': 'Heaven'
+        }
+    )
+
+    new_employee_entity = request.execute()
+
+
+Batch requests
```

Comment:
I'll rather see batch requests in separate section/page. Or move example that contains modifications to creation/modification section.